### PR TITLE
Fits format plugin

### PR DESCRIPTION
**Pull request by blink1073**
We need to add the FITS files from 
https://github.com/scikit-image/scikit-image/tree/master/skimage/data to
https://github.com/imageio/imageio-binaries/tree/master/images

to facilitate testing.  Shall I submit a PR for that?

### CODE REVIEW

**Comment by blink1073** [comment]
While I'm adding files, it would be nice to add the TIFF files for the same reason.

**Comment by almarklein** [comment]
You mean to be able to test the tiff plugin?

**Comment by blink1073** [comment]
We have more extensive tests in skimage that rely on those files.

**Comment by almarklein** [comment]
I've given you commit rights, so you can more easily push files to that repo. (Naturally, all changes to imageio/imageio should go via PR).

**Comment by blink1073** [comment]
mwahaha!

File: imageio/plugins/fits.py [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+    from astropy.io import fits as _fits
+    return _fits
+
+
+class FitsFormat(Format):
+
+    """ Flexible Image Transport System (FITS) is an open standard defining a
+    digital file format useful for storage, transmission and processing of
+    scientific and other images. FITS is the most commonly used digital
+    file format in astronomy.
```
Comment (almarklein) [inline imageio/plugins/fits.py]: Should we mention that astropy is needed to use read/write images of this format?

**Comment by blink1073** [comment]
All set.  I only added read support since that's all we had in scikit-image.

**Comment by almarklein** [comment]
The tests should run on the python 2.7 build, correct? It seems they're skipped.

**Comment by blink1073** [comment]
The tests are run without optional dependencies, then with them (that's what we did in scikit-image).

**Comment by almarklein** [comment]
Should it not be enough to run the tests *with* optional dependencies on one build, and without in all the others?

**Comment by blink1073** [comment]
Your call brother. 



—

**Comment by almarklein** [comment]
I prefer one run :)

**Comment by blink1073** [comment]
I'm getting this error all of a sudden:

 InternetNotAllowedError: Will not download resource from the internet because enironment variable IMAGEIO_NO_INTERNET is set.

**Comment by almarklein** [comment]
Ah doh. When the test is run with `installed`, internet access is restricted.

**Comment by blink1073** [comment]
I fixed it within the test.

**Comment by almarklein** [comment]
Euhm, so I suppose we should run the test with astropy in another env (not the 2.7 one).

**Comment by blink1073** [comment]
Ah, I see that it will skip the test entirely, ha!

**Comment by blink1073** [comment]
What to create a specific build for optional dependencies?

**Comment by blink1073** [comment]
Or just target 3.4?

**Comment by almarklein** [comment]
Yes, 3.4 is sort of the "main"

**Comment by blink1073** [comment]
Okay, done.

**Comment by almarklein** [comment]
Great, thanks!

**Pull request merged**